server/bundler/loader.js

```diff
@@ -12,7 +12,6 @@ function getSectionsModule( sections ) {
 			"\tlayoutFocus = require( 'lib/layout-focus' ),",
 			"\tReact = require( 'react' ),",
 			"\tLoadingError = require( 'layout/error' ),",
-			"\tclasses = require( 'component-classes' ),",
```

Comment:
Curious why this didn't require more changes, but looks like the dependency was left erroneously (6224-gh-calypso-pre-oss).